index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+
+    <div id="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+        </div>
+
+        <div id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
```

Comment:
IMG element should have an Alternative text using the property `alt=""`

Reply:
- [ ] And again I strongly suggest you to write comments politely and add a brief explanation 🤗.

Reply:
Thank you! Will try again :smile: